assets/chezmoi.io/docs/reference/commands/add.md

```diff
@@ -88,6 +94,8 @@ directory, create a symlink template with `.chezmoi.sourceDir` or
     See [this GitHub issue](https://github.com/twpayne/chezmoi/issues/1574) for
     details.
 
+## Example
```

Comment:
I don't know whether to aim for consistency here or accuracy. Strictly speaking, this should be `## Examples` because there are five example commands. It's using *one* `!!! example` block, so it *could* be `## Example` as it is now, but that doesn't feel like correct grammatical agreement to me. In the commands where we only have one example, would `## Example` be correct, or should we *always* use plural forms  for these section headers (e.g., `## Flags`, `## Common flags`, `## Notes`, `## Examples`).

I’m leaning toward consistency on section header plurals, even if it means that there's no longer section body grammatical number agreement.

Reply:
Let's change to plural.